nix/profile.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, writeScriptBin, gnutar, kore-exec-prof, kore-exec-infotable, hp2pretty, hs-speedscope, eventlog2html }:
+
+writeScriptBin "profile" ''
+  #! ${stdenv.shell}
+  filename=$(basename -- "$1")
+  timeout="''${2:-4m}"
+
+  mkdir -p profile
+  ${gnutar}/bin/tar -xzvf "$1" -C ./profile
+  cd profile
+  # generate .prof and .eventlog profiles
+  timeout --foreground -s INT "$timeout" env GHCRTS='-p -l-au' PATH='$PATH:${kore-exec-prof}/bin' ./kore-exec.sh
+  mkdir -p ../profile-$filename/raw
```

Comment:
(nit) Maybe factor out `profile-$filename`? (and create it in the beginning, together with `./profile`)

Reply:
Should we also include a timestamp in the directory name? Would make it easier to manage the result files if we store them in a custom place (cleaning up based on time, avoiding overwrites)

Reply:
good idea, I've added a timestamp